keps/sig-apps/4017-pod-index-label/README.md

```diff
@@ -411,7 +411,7 @@ existing label which other things may depend on, for example).
 - Docs are clear about what happens if two pods get the same value (it is set by workload controllers, nothing in the API system will prevent collisions from happening).
 
 #### GA
```

Comment:
Please add a point about 
Locking the `PodIndexLabel` feature-gate, and modifying the code to ignore it

Reply:
Added, PTAL

Reply:
I don't see a change here.

Reply:
missed pushing it, should be there now

Reply:
thanks, lgtm